formats/application/vnd/oada/isoblue/heartbeat/1/+json/schema.js

```diff
@@ -0,0 +1,67 @@
+/* Example:
```

Comment:
This example should go in examples/default.js rather than be a comment at the top of the schema file.